### PR TITLE
path config->home + prompt to auto add

### PR DESCRIPTION
**Pull request by anhad13**
Changed default for export file and added option to auto add to bashrc.
Downside: it adds only the general project and data dirs to bashrc, not any new ones that we may come up with.

### CODE REVIEW

**Comment by pruksmhc** [comment]
Were you able to figure out if this works on Mac OSX?

**Comment by anhad13** [comment]
Yes, will get the PR ready soon.

**Comment by sleepinyourhat** [comment]
Try to get this in ASAP. Also make sure to add a note in the readme explaining what to set up and how.

**Comment by anhad13** [comment]
Tested on Mac (my system) and Linux (Prince). Should be ready for a look.

**Comment by sleepinyourhat** [comment]
Thanks! Three things:

- We don't want people to be editing the file called `path_config.sh`, since having an edited file in the repo directory causes all kinds of chaos with git. Everyone at JSALT, at least, made a copy of `path_config.sh` called `user_config.sh`, which is automatically in `.gitignore`, and edited that. Mind updating the script to refer to `user_config.sh`, and making sure that the readme and your script and path_config.sh all make it clear that that's the thing to do. It might also make sense to rename `path_config.sh` to `user_config_template.sh`.
- Move the script to `scripts/`.
- Fix merge conflicts.

**Comment by sleepinyourhat** [comment]
We need this tonight! Mind updating the new tutorial to match, as well?

http://socrates.io/#O5j4e3H

**Comment by anhad13** [comment]
Changes fixed. WIll update tutorial now!

File: scripts/export_from_bash.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Running 'source scripts/export_from_bash.sh' will add the job of running path_config.sh directly to bash
+# (bashrc for Linux and bash_profile for Mac.)
+# for automatic updates on booting up the terminals.
+
+function changebashpaths() {
+	output="source $(pwd)/user_config_template.sh"
```
Comment (sleepinyourhat) [inline scripts/export_from_bash.sh]: Should be user_config.sh!

**Comment by sleepinyourhat** [comment]
Had to do a bit more cleanup myself—short on time. Merging.

**Pull request merged**